packages/components/src/placeholder/style.scss

```diff
@@ -1,4 +1,5 @@
-.components-placeholder.components-placeholder { // This needs specificity to override individual block styles.
+.components-placeholder.components-placeholder {
+	// This needs specificity to override individual block styles.
```

Comment:
I believe the comment here is in the wrong place. Maybe try moving it above the selector?

Reply:
Good point. I believe this change happened with auto-formatting when saving the file.

Addressed in https://github.com/WordPress/gutenberg/pull/31713/commits/030c1bfeab295f402867a3a419e624855f367e0c